### ISSUE TITLE
Cache gifs

### ISSUE THREAD

**Issue by joshghent** [joshghent/gifbar#4]
Cache the GIFs that are returned in recent or on search

**Comment by hiteshpr** [comment]
I can work on this.

**Comment by joshghent** [comment]
@hiteshpr All your my friend! Thank you for offering to help 👍 I look forward to your PR

**Comment by joshghent** [comment]
https://medium.com/@philipp.schaechtele/asset-caching-with-service-worker-c40dcda43842

**Comment by Samar1110** [comment]
@joshghent I am new to open source, could you allow me to contribute on this project.

**Comment by abhi-shek01** [comment]
Hi @joshghent, Is it still open? I would like to contribute.

**Comment by joshghent** [comment]
> Hi @joshghent, Is it still open? I would like to contribute.

Of course! I think the project is quite old though so largely needs a lot of love to upgrade it's dependencies. I'd appreciate any support :) Happy to help if you have any issues.
